docs/operation/volume_management/volume_creation_deletion_cli.rst

```diff
@@ -0,0 +1,83 @@
+Volume Management using the CLI
+===============================
+
+To use persistent storage in a MetalK8s cluster, one needs to create Volume
+objects.
+In order to create Volumes you need to have StorageClass objects registered in
+your cluster. See :doc:`/operation/volume_management/storageclass_creation`
+
+Volume Creation
+---------------
+
+This section describes how to create a **Volume** from the **CLI**.
```

Comment:
More specifically, a `Volume` spanning an entire block device.

Reply:
Could be controversial since we support `spareLoopDevice` and `rawBlockDevice`

let us keep it simple.